### PR TITLE
Multi region Infrastructure outputs

### PR DESCRIPTION
**Pull request by geekbass**
Need the following Infrastructure outputs for Multi Region Module. Submitting current Branch as PR and behalf of @bernadinm .

### CODE REVIEW

File: docs/published/main.tf [hunk 1]
```diff
@@ -11,16 +7,30 @@ data "http" "whatismyip" {
...
+  region = "us-east-1"
+}
+
+provider "aws" {
+  region = "us-west-2"
+  alias = "spoke-1"
+}
+
 module "dcos" {
+<<<<<<< HEAD
```
Comment (fatz) [inline docs/published/main.tf]: conflict leftovers. Please have a look

File: docs/published/main.tf [hunk 2]
```diff
@@ -11,16 +7,30 @@ data "http" "whatismyip" {
...
+  source  = "git@github.com:dcos-terraform/terraform-aws-dcos?ref=multi-region"
+  # version = "~> 0.1.0"
+=======
   source  = "dcos-terraform/dcos/aws"
   version = "~> 0.1.0"
+>>>>>>> 4fa09ef86efda42653994017504c367da5bdea24
```
Comment (fatz) [inline docs/published/main.tf]: conflict leftover

File: variables.tf [hunk 3]
```diff
@@ -215,3 +215,9 @@ variable "cluster_name_random_string" {
   description = "Add a random string to the cluster name"
   default     = false
 }
+
+variable "internal_networks" {
+  description = "Subnet ranges for all internal networks"
+  type        = "list"
+  default     = []
+}
```
Comment (bernadinm) [inline variables.tf]: @geekbass and @fatz with this addition of the `internal_networks` will this no longer require multiple `terraform apply`?

File: docs/published/main.tf [hunk 4]
```diff
@@ -47,4 +47,4 @@ output "cluster-address" {
 
 output "public-agents-loadbalancer" {
   value = "${module.dcos.public-agents-loadbalancer}"
-}
+}
```
Comment (fatz) [inline docs/published/main.tf]: please always have newline at the end of file @geekbass

File: variables.tf (hunk 3 above)
Comment (fatz) [reply variables.tf]: indeed

**Pull request merged**